1_uprava.js

```diff
@@ -0,0 +1,8 @@
+const countVowels = (text) => {
+    return text.match(/[aeiou]/gs).length;
```

Comment:
Tu si mal zadefinovanu konstantu count ktoru si nasledne vracal, vies to spravit aj takto inlinovo, lepsie sa to cita a opat, cistejsi kod.
Z:
```ts
    const count = text.match(/[aeiou]/gs).length;
    return count;
```
Zjednodusis na:
```ts
    return text.match(/[aeiou]/gs).length;
```